scene/resources/syntax_highlighter.h

```diff
@@ -139,6 +141,8 @@ class CodeHighlighter : public SyntaxHighlighter {
 
 	void set_member_variable_color(Color p_color);
 	Color get_member_variable_color() const;
+
+	void set_uint_suffix_enabled(bool p_enabled = true);
```

Comment:
I'm curious as to why this setter has a default parameter. We generally don't use default parameters for setter methods by convention.

Reply:
I just followed @Chaosus 's suggestion for this, in this case looks like explicit is better